editor/doc_tools.cpp

```diff
@@ -382,7 +382,7 @@ void DocTools::generate(BitField<GenerateFlags> p_flags) {
 				continue;
 			}
 
-			String cname = name;
+			const String &cname = name;
```

Comment:
Same here

Reply:
The function body this snippet is located at is about 683 lines long. The `cname = <something>` pattern is used in quite a few places. I think it adds patterning and readability for the maintainer.